src/pages/Overview/SchoolNav/SchoolNav.jsx

```diff
@@ -1,26 +1,88 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import './SchoolNav.css';
+import { Container } from 'react-bootstrap';
+import Signin from '../../../components/Signin';
+import { AuthProvider } from '../../../contexts/AuthContext';
+
+function LoginView(props) {
+  const { school } = props; // destructuring prop or else would get linter error
+  return (
+    <>
+      <h3>{school} is using a virtual textbook exchange!</h3>
+      <h5>To buy or sell textbooks, please login or create an account below.</h5>
+
+      <AuthProvider>
+        <Container
+          className="d-flex align-items-center justify-content-center"
+          style={{ minHeight: '40vh' }}>
+          <div className="w-100" style={{ maxWidth: '400px' }}>
+            <Signin />
+          </div>
+        </Container>
+      </AuthProvider>
+    </>
+  );
+}
+
+function ProdOneDesc(props) {
+  const { school } = props; // destructuring prop or else would get linter error
+  return (
+    <>
+      <h3>{school} is using an inperson textbook exchange!</h3>
+      <h5>
+        To buy or sell textbooks, you must go inperson to the {school} Textbook Exchange. Please
+        reach out to your student government to learn more about the open times for
+        {school}&apos;s in-person textbook exchange. You may browse the textbooks available in the
+        &apos;browse&apos; tab above.
+      </h5>
+    </>
+  );
+}
 
 function SchoolNavPage() {
+  const schools = [
+    'Tufts University',
+    'Wesleyan University',
+    'Northeastern University',
+    'Tower Hill School'
+  ];
+
+  const [selectedSchool, setSelectedSchool] = React.useState('none');
+
+  // Will use this list in future versions
+  /* const schoolList = [
+    ['Tufts University', 'virtual'],
+    ['Wesleyan University', 'in-person'],
+    ['Northeastern University', 'virtual'],
+    ['Tower Hill School', 'in-person']
```

Comment:
note this will probably have to change. We will want a different data structure setup

Reply:
Im thinking something like 

```json
{
  "in-person": [
    {
      "id": "2345-1234-123"
      "name": "Tufts"
    },
    ...
  ],
  "virtual": [...]
}
```

Reply:
Good point! Agreed! Will update in my next pull request!